File: program1_1.java
// Program to divide two numbers

import java.util.*;

class DivideX
{
    public int iNo1, iNo2;

    public DivideX(int iValue1, int iValue2)
    {
        this.iNo1=iValue1;
        this.iNo2=iValue2;
    }

    public int Devide()
    {
        int iAns= iNo1/iNo2;
        return iAns;
    }
}

class program1_1
{
    public static void main(String Arg[])
    {
        Scanner sobj= new Scanner (System.in);
        int iRet=0;

        System.out.println("Enter First Number:");
        int iValue1=sobj.nextInt();

        System.out.println("Enter Second Number:");
        int iValue2=sobj.nextInt();

        DivideX dobj = new DivideX(iValue1,iValue2);
        iRet=dobj.Devide();

        System.out.println("Division is :"+iRet);



    }
}

File: Program1.1.c
/////////////////////////////////////////////////////////////////
//  Problem statement:Program to divide two numbers.
//  Input:
//  Output:
/////////////////////////////////////////////////////////////////
#include<stdio.h>

int Divide(int iNo1, int iNo2)
{
    int iAns=0;

    if(iNo2<0)
    {
        return -1;
    }

    iAns=iNo1/iNo2;

    return iAns;
}
int main()
{
    int iValue1=15;
    int iValue2=5;
    int iRet=0;

    iRet=Divide(iValue1,iValue2);

    printf("Division is %d",iRet);

    return 0;

}